### PR TITLE
Fix "matchExpressions" for "nodeAffinity"

### PR DESCRIPTION
**Pull request by chs2**

### CODE REVIEW

File: docs/kubernetes.rst
Comment (eladkal) [inline docs/kubernetes.rst:1]: Is this doc refrenced somewhere? It seems old and outdated (it even imports from contrib)
cc @RNHTTR  I assume we might need a followup to https://github.com/apache/airflow/pull/37728 to handle this doc?

**Comment by amoghrajesh** [comment]
@chs2 can you please update the PR description? Its hard to tell what you are trying to achieve without a description

**Review by hussein-awala — APPROVED** [review]
`matchExpressions` should be an array of expressions, LGTM

File: docs/kubernetes.rst
Comment (RNHTTR) [reply docs/kubernetes.rst:1]: I don't think so. It seems correct in the [docs for the current version of the provider](https://airflow.apache.org/docs/apache-airflow-providers-cncf-kubernetes/stable/operators.html#how-to-use-cluster-configmaps-secrets-and-volumes-with-pod) -- this issue is for the [KPO docs in 1.10.15](https://airflow.apache.org/docs/apache-airflow/1.10.15/kubernetes.html#kubernetespodoperator)

**Review by amoghrajesh — APPROVED** [review]

**Comment by amoghrajesh** [comment]
@chs2 are you working on this actively? Requires small nit handling and a rebase (doing it)

**Comment by amoghrajesh** [comment]
Reopened to run the latest set of CI on the PR

**Comment by chs2** [comment]
> @chs2 are you working on this actively? Requires small nit handling and a rebase (doing it)

To be true, no I'm not.
I wouldn't have thought it would raise so many questions.
While working on a airflow1-to-airflow2 migration I found the [Airflow1 KubernetesPodOperator documentation](https://airflow.apache.org/docs/apache-airflow/1.10.15/kubernetes.html#kubernetespodoperator). I noticed the typo and thought it would be nice to have it fixed for anyone in the same situation.

**Comment by eladkal** [comment]
Ah wait
This PR is targeted the v-10-stable branch. This is why it was confusing.
We are not having any more releases for Airflow 1.10

**Pull request closed without merging**